src/components/App/App.tsx

```diff
@@ -1,8 +1,60 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Route, Switch, NavLink } from 'react-router-dom';
 import { Header } from '../Header/Header';
+import mockPosts from '../../mockdata/mockPosts.json';
+import { Cards } from '../Cards/Cards';
 
-interface AppProps {}
+export interface Post {
+  type: string;
+  id: number;
+  attributes: {
+    user_id: number;
+    username: string;
+    title: string;
+    description: string;
+    photo: string;
```

Comment:
Not sure if this matters, but in our DB "photo" is called "image" for a post

Reply:
I'm gonna work on hooking into the posts BE today. I have a feeling this whole structure will change, but I will change to image in the mockData for now